ee/clickhouse/views/events.py

```diff
@@ -144,19 +144,20 @@ def values(self, request: Request, **kwargs) -> Response:  # type: ignore
                     flattened.append(value[0])
         return Response([{"name": convert_property_value(value)} for value in flatten(flattened)])
 
-    @action(methods=["GET"], detail=False)
-    def sessions(self, request: Request, *args: Any, **kwargs: Any) -> Response:  # type: ignore
-        filter = SessionsFilter(request=request, team=self.team)
+    # TODO: DEPRECATED
+    # @action(methods=["GET"], detail=False)
```

Comment:
Unsure about this change. Should we keep the API endpoints? If the answer is no, should we remove all the filtering, queries, .... ? Any thoughts @macobo @rcmarron @alexkim205

Reply:
If it's dead then it should be killed. However @rcmarron owns this.

Reply:
Are you in a rush to get this merge or can we wait for Rick to be online?

Reply:
Alright, I've removed all the legacy code from the backend as well but will wait for @rcmarron's review before merging as I'm removing a lot of stuff